src/libcommon/utility/types.h

```diff
@@ -344,7 +346,8 @@ typedef enum {
     IoErrorIsADirectory,
     IoErrorNoSuchFileOrDirectory,
     IoErrorResultOutOfRange,
-    IoErrorUnknown
+    IoErrorUnknown,
```

Comment:
Don't you think `Unknown` should be the first or the last value?

Reply:
Yes, it think it should. Moved it in commit: https://github.com/Infomaniak/desktop-kDrive/pull/43/commits/d31509858445aca8c91dca224e75c3ebd730993c